src/mcom/integration.md

```diff
@@ -0,0 +1,24 @@
+---
+ee_only: true
+title: Integration
+---
+
+Integration enables you to add or disable the Order Management System (OMS) integration for Magento Commerce (MC) and view details of the integration.
+
+![Integration]({% link images/images-ee/mcom-integration.png %}){: .zoom}
+_Integration_
+
+## Register the integration
+
+1. On the _Admin_ sidebar, go to **MCOM** > **Integration**.
+1. Click **Register** to register the MCOM (also known as OMS) integration.
+
+## Disable or enable the integration
+
+1. On the _Admin_ sidebar, go to **MCOM** > **Integration**.
+1. Check the status of the integration in the _Status_ row---either `Enabled` or `Disabled`.
+   1. To enable a disabled integration, click **Enable**. A confirmation message appears to inform you the integration has been enabled.
+   1. To disable an enabled integration, click **Disable**. A confirmation message appears to inform you the integration has been disabled.
```

Comment:
```suggestion
   1. To disable an enabled integration, click <span class="btn">Disable</span>. A confirmation message appears to inform you the integration has been disabled.
```

Reply:
I really think the span button makes things busier than they need to be, and is a totally unneeded formatting convention. I will revise them to be span buttons, but think this style decision should be reevaluated, as I know that not everyone follows this for the same reason.

Reply:
Also, isn't HTML something we try not to use at all, ever?